images/win/Windows2022-Archive_Grouped.json

```diff
@@ -0,0 +1,1594 @@
+[
+  {
+    "Enabled windows optional features": [
+      {
+        "OSVersion": "OS Version: 10.0.20348 Build 1129"
+      },
+      {
+        "Image Version": "Image Version: "
+      }
+    ]
+  },
+  {
+    "Language and Runtime": [
+      {
+        "WSL": "Windows Subsystem for Linux [WSLv1]"
+      }
+    ]
+  },
+  {
+    "Package Management": [
+      {
+        "Bash": "Bash 5.1.16(1)-release"
+      },
+      {
+        "Go": "Go 1.17.13"
+      },
+      {
+        "Julia": "Julia 1.8.2"
+      },
+      {
+        "LLVM": "LLVM 14.0.6"
+      },
+      {
+        "Node": "Node 16.18.0"
+      },
+      {
+        "Perl": "Perl 5.32.1"
+      },
+      {
+        "PHP": "PHP 8.1.11"
+      },
+      {
+        "Python": "Python 3.9.13"
+      },
+      {
+        "Ruby": "Ruby 3.0.4p208"
+      },
+      {
+        "Kotlin": "Kotlin 1.7.20"
+      }
+    ]
+  },
+  {
+    "Package Management | Environment variables": [
```

Comment:
This header points to the env variables but the section doesn't contain them.

Reply:
Thanks for pointing it out,
corrected